iceoryx_dds/test/CMakeLists.txt

```diff
@@ -13,15 +13,26 @@
 # limitations under the License.
 
 cmake_minimum_required(VERSION 3.5)
-project(test_iox_to_dds VERSION 0)
+# project(test_iox_to_dds VERSION 0)
```

Comment:
Hmm, all test projects have their version set to 0, the reason why escapes me though.
@elBoberido Can you shed some light on this ?